js-miniapp-sdk/README.md

```diff
@@ -81,6 +81,24 @@ miniApp.requestLocationPermission()
 	});
 ```
 
+### 4. Show Interstitial Ad
+
+This is a helper function to make request to js bridge to show Iterstitial ads on the native SDK, The response is captured by a specific response type that is called InterstitialAdResponse and can be found here `src/types/responseTypes/interstitial/index.ts`.
```

Comment:
```suggestion
This is a helper function to make request to js bridge to show Iterstitial ads on the native SDK, The response is captured by a specific response type that is called InterstitialAdResponse and can be found [here](api/interfaces/interstitial.html).
```

The userguide and generated docs are published [here](https://rakutentech.github.io/js-miniapp). So if you need to link to a type, it should point to the generated docs. You can check the generated documentation by running `yarn sdk docs` and then look at the `js-miniapp-sdk/publishableDocs` folder.

Reply:
Updated the doc !
Thanks for sharing the tip!